glide.yaml

```diff
@@ -31,8 +31,10 @@ import:
 - package: github.com/spaolacci/murmur3
```

Comment:
do we need both this and the stack version?

Reply:
One is faster for writing, and the other is faster for reading concurrently hehe (stack based approach needs no hash object pooling and hence no read/write locks, but is slightly slower in constant time - when we write out the bloom filter its sequential so no need to for concurrent hashing).

Reply:
Sigh, lol.